src/greenlight/test.clj

```diff
@@ -173,12 +196,12 @@
 
 (defn run-test!
   "Execute a test. Returns the updated test map."
-  [system test-case]
+  [system options test-case]
```

Comment:
🙅‍♂️ This is an incompatible change to a public function. It is used in at least one other project I know of ([caioaao/kaocha-greenlight](https://github.com/caioaao/kaocha-greenlight/blob/master/src/caioaao/kaocha_greenlight/test/var.clj)) which would be broken by this. Instead, we should preserve the old arity and have it substitute an empty map for `options` in that case.